### PR TITLE
add more pillows

### PR DESCRIPTION
**Pull request by emord**
@gcapalbo @dannyroberts to be in for deploy

### CODE REVIEW

File: fab/pillows/production.yml [hunk 1]
```diff
@@ -5,27 +5,21 @@ exclude_groups:
...
   - custom.opm.models.OpmUserFluffPillow
   - custom.reports.mc.models.MalariaConsortiumFluffPillow
-#   - custom.m4change.models.AncHmisCaseFluffPillow
-#   - custom.m4change.models.LdHmisCaseFluffPillow
-#   - custom.m4change.models.ImmunizationHmisCaseFluffPillow
-#   - custom.m4change.models.ProjectIndicatorsCaseFluffPillow
-#   - custom.m4change.models.McctMonthlyAggregateFormFluffPillow
-#   - custom.m4change.models.AllHmisCaseFluffPillow
+  - custom.m4change.models.M4ChangeFormFluffPillow
 #   - custom.intrahealth.models.CouvertureFluffPillow
```
Comment (emord) [inline fab/pillows/production.yml:11]: Going to squash these into 1 as soon as I figure out the tests

File: fab/pillows/production.yml [hunk 2]
```diff
@@ -5,27 +5,21 @@ exclude_groups:
 # include_groups:
 #   - mvp_indicators
 include_pillows:
-#   - custom.bihar.models.CareBiharFluffPillow
```
Comment (emord) [inline fab/pillows/production.yml:8]: Removing since this should only be in india

File: fab/pillows/production.yml [hunk 3]
```diff
@@ -5,27 +5,21 @@ exclude_groups:
...
   - custom.opm.models.OpmUserFluffPillow
   - custom.reports.mc.models.MalariaConsortiumFluffPillow
-#   - custom.m4change.models.AncHmisCaseFluffPillow
-#   - custom.m4change.models.LdHmisCaseFluffPillow
-#   - custom.m4change.models.ImmunizationHmisCaseFluffPillow
-#   - custom.m4change.models.ProjectIndicatorsCaseFluffPillow
-#   - custom.m4change.models.McctMonthlyAggregateFormFluffPillow
-#   - custom.m4change.models.AllHmisCaseFluffPillow
+  - custom.m4change.models.M4ChangeFormFluffPillow
```
Comment (emord) [inline fab/pillows/production.yml:10]: The checkpoint for this should be set to 49852429 after deploy (minimum of the other checkpoints) I'll do this

**Review by dannyroberts — APPROVED** [review]
Looks good. Thanks for the careful thought into these PRs. I think sharing kafka confumers like this has a good chance of speeding things up

**Pull request merged**